src/concepts/segments.md

```diff
@@ -0,0 +1,134 @@
+# Segments
+
+A segment is a group of clients who fit a set of criteria at a point in time. 
+The set of criteria itself can also be referred to as a "segment".
+
+Typically you'll use segments to gain more insight into what is going on, by asking 
+"Regarding the thing I'm interested in, 
+do users in different segments behave differently, 
+and what insights does this give me into the users and the product?" 
+For example, "In this experiment, how do new users react to this feature, 
+and how does this differ from established users?". 
+Or "DAU moved dramatically - 
+is this restricted to users in this particular country (i.e. a segment) 
+where there's an event happening, 
+or is it global and therefore not solely due to that event?"
+
+## Versioning
+
+We are building out our library of segments, 
+and we want room to iterate to improve them in the future. 
+So please quote segments' versions with their names, 
+so that your communication is forwards compatible.
+
+## Current segments
+
+### Regular users v1, Semi-regular users v1, New/irregular users v1
+
+This is a set of three segments. 
+On a given day, every client falls into exactly one of these segments.
+Each client's segment is stored in `telemetry.clients_last_seen.segment_regular_users_v1`.
+(FIXME: or it will be if the [PR](https://github.com/mozilla/bigquery-etl/pull/825) is accepted in a form similar to its current state)
+
+Regular users v1 is defined as 
+clients who have at least two days where they browsed >=5 URIs, 
+in *all four* of the last weeks. 
+Observationally, on any given day this segment seems to contain a large fraction of our users 
+and has exceptionally high retention.
+
+Semi-regular users v1 is defined as 
+clients who browsed >=5 URIs on at least two days in each of *one, two or three* of the last four weeks. 
+This seems to be a smaller segment of our users, and has relatively high retention.
+
+New/irregular users v1 is defined as 
+clients who browsed >=5 URIS on at least two days in *none* of the last four weeks. 
+This is a smaller segment of our users and has low retention 
+(though "activation" is likely a more relevant word than "retention" for many of these clients).
+
+In each of these definitions, 
+"the last four weeks" refer to 7 day periods that end on the day of week 
+before the first day for which we want to study the user. 
+They do not necessarily run Monday through Sunday.
+
+## Obsolete segments
+
+None yet.
+
+## Writing queries with segments
```

Comment:
I think we should at least mention somewhere the potential problem that a ping with a given submission_date may be talking about activity that happened on an earlier date.

Reply:
Can you give a pathological example where that would mislead someone in a segment-specific way?

Reply:
If the user uses the browser in one week, closes the browser before a ping is sent, and then reopens a week later for further use, we'll think that the user was inactive in the first week.

Reply:
Yep, this is a general hazard of using `submission_date` - although vastly reduced by [pingsender](https://firefox-source-docs.mozilla.org/toolkit/components/telemetry/internals/pingsender.html). Since it's a general hazard, it could be mentioned on every page on DTMO that uses `submission_date` - and there are lots of other hazards that could be mentioned on every page too. This is impractical, so let's leave this to the "Common Analysis Gotchas" page that defines `submission_date`, mentions that using it (rather than client timestamps) is the standard for our data analysis and links to why.

I think this is the kind of detail where the people for whom it's going to be an issue (DSes forensically tracking down why numbers are slightly different than expected) should be expected to know about the properties of `submission_date`, and they are likely to expect that a behavioural segment defined in terms of past usage is defined in terms of `submission_date`, with all the consequences that entails. And if they don't expect it, they can track it down in the ETL.

If this was going to mislead stakeholders as they try to use these segments to do things, then yes we should flag those issues. But I can't think of likely/plausible ways for this to happen...

Reply:
OK, fair enough :)